services/slam/slam.go

```diff
@@ -124,8 +124,8 @@ func InternalStateFull(ctx context.Context, slamSvc Service) ([]byte, error) {
 }
 
 // Limits returns the bounds of the slam map as a list of referenceframe.Limits.
-func Limits(ctx context.Context, svc Service) ([]referenceframe.Limit, error) {
-	data, err := PointCloudMapFull(ctx, svc)
+func Limits(ctx context.Context, svc Service, useEditedMap bool) ([]referenceframe.Limit, error) {
```

Comment:
named this `useEditedMap` instead of  `returnEditedMap` because the function does not return a map